src/app/common/button/button.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="button" #button
+     [ngStyle]="styles.elStyle"
```

Comment:
ne razumem zasto si ovo radio sa ngStyle-om?, lepo ti napisah na mesendzeru da iskoristis ngClass

`[ngClass]="{'first': true, 'second': true, 'third': false}"` kao sto vidis mozes da dodas komponenti bezbroj boolean propertija i bezbroj css klasa

Reply:
npr ovde stavis `[ngClass] = {'is-full': isFull}`